certbot-dns-google/setup.py

```diff
@@ -0,0 +1,70 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+version = '0.15.0.dev0'
+
+# Please update tox.ini when modifying dependency version requirements
+install_requires = [
+    'acme=={0}'.format(version),
+    'certbot=={0}'.format(version),
+    'fasteners',
+    'google-api-python-client',
+    'mock',
```

Comment:
Since we import it directly, I think it's good practice to list `oauth2client` as a dependency.